doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -170,6 +170,7 @@ nRF9160: Serial LTE modem
 * Added:
 
   * AT command ``#XWIFIPOS`` to get Wi-Fi location from nRF Cloud.
+  * Support of *WRITE REQUEST* in TFTP client.
```

Comment:
```suggestion
  * Support for write requests in the TFTP client.
```

Reply:
I guess that's fine, then. But I think our release notes are the only place where we use the full capitalized version.